UI/GalaxySetupWnd.cpp

```diff
@@ -764,18 +764,8 @@ void GalaxySetupPanel::CompleteConstruction() {
     SettingsChangedSignal();
 }
 
-namespace {
-    // set of characters from which to generate random seed that excludes some ambiguous letter/number pairs
-    static char alphanum[] = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
-}
-
 void GalaxySetupPanel::RandomClicked() {
-    std::string s;
-    ClockSeed(); // to ensure we don't always get the same sequence of seeds
-    for (int i = 0; i < 8; ++i)
-        s += alphanum[ RandSmallInt(0, (sizeof(alphanum) - 2))];
-    m_seed_edit->SetText(s);
-    //std::cout << "GalaxySetupPanel::RandomClicked() new seed: " << s << std::endl;
+    m_seed_edit->SetText("RANDOM");
```

Comment:
doesn't work in languages other than english

Reply:
TY, addressed by storing it separate.
Note an actual seed is always displayed in mp lobby.  I did not address this so players can compare if they suspect different seeds (or want to argue about why one char should change ;) ).